test-data/unit/check-newtype.test

```diff
@@ -310,17 +310,15 @@ main:4: error: Invalid type "__main__.T"
 from typing import NewType
 
 a = 3
-a = NewType('a', int)
+def f(): a
```

Comment:
Allowing redefinition with a different kind of symbol looks dangerous from the point of view of fine grained incremental, but maybe it is OK.